src/dataService/savedItemsService.ts

```diff
@@ -37,8 +39,11 @@ export class SavedItemDataService {
     [SavedItemStatus.DELETED]: 'DELETED',
     [SavedItemStatus.HIDDEN]: 'HIDDEN',
   };
-  private readDb: Knex;
+  private db: Knex;
+  private writeDb: Knex;
```

Comment:
so we dump the read and write client into the constructor through the context when we create a savedItemsService.